FWCore/Common/src/LuminosityBlockBase.cc

```diff
@@ -78,15 +75,17 @@ namespace edm
             }
 
             std::pair<TriggerNamesMap::iterator, bool> ret =
-               triggerNamesMap.insert(std::pair<edm::ParameterSetID, edm::TriggerNames>(triggerResults.parameterSetID(), triggerNames));
+               triggerNamesMap.insert(std::pair<edm::ParameterSetID,
+   edm::TriggerNames>(triggerResults.parameterSetID(), triggerNames));
```

Comment:
Nice that it split the long line, but I would consider the new indentation here to be erroneous.